src/js/actions/signalActions.ts

```diff
@@ -13,13 +15,7 @@ var ns = require('../util/ns'),
  * @param {*} value - The initial value of the signal
  * @returns {Object} An action object
  */
-function initSignal(signal, value) {
-  return {
-    type: INIT_SIGNAL,
-    signal: ns(signal),
-    value: value
-  };
-}
+export const initSignal = createStandardAction('INIT_SIGNAL')<any, string>();
```

Comment:
Like the other action files, could you add a comment to these lines to indicate what the two parameters are. E.g., I'm thinking `// initial value, signal name`